tools/ExtensionsMetadataGenerator/test/ExtensionsMetadataGeneratorTests/ExtensionsMetadataGeneratorTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Linq;
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace ExtensionsMetadataGeneratorTests
+{
+    public class ExtensionsMetadataGeneratorTests
+    {
+        [Fact]
+        public void GenerateExtensionReferences_Succeeds()
```

Comment:
These are tests. They test things. Tests are good. All our code should have tests :)